recipes/vcpkg-tool/recipe.yaml

```diff
@@ -0,0 +1,46 @@
+context:
```

Comment:
We don't support these here 😛 Can you leave this out for now?

Reply:
sure, I wanted to remove that after checking that the build works.